dash_html.py

```diff
@@ -408,58 +422,70 @@ def set_html(app):
     )
 
 
+def no_solution(num_vehicles: int) -> tuple[html.H5, html.Table]:
+    """UI feedback when no solution is found."""
+    column_count = 2 + len(RESOURCES)
+    values_dicts = {
+        vehicle + 1: {index: "---" for index in range(column_count)}
+        for vehicle in range(num_vehicles)
+    }
+    values_totals = ["---" for _ in range(column_count)]
+
+    return (
+        html.H5(className="results no-results", children=["no solution found"]),
+        create_table(values_dicts, values_totals),
+    )
```

Comment:
Yeah I went back and forth as to whether to move this over but I did because I figured that way any potential config changes (location, bounds, etc) would support a no solution option but if you think it's overkill I can get rid of it. I'm very on the fence.

Reply:
I actually think the only solver that even could fail this way is NL (the others always fill the `solution_cost` variable with something and will never call this function), and if it does, it raises an uncaught exception. So thinking about it, the way the code is written right now, I suspect that `no_solution` is never called, no matter the parameters. 🤔

Reply:
Should I remove it then?

Reply:
Sure, let's re-add it if required with other changes later.

Reply:
removed: https://github.com/dwave-examples/mvrp/pull/11/commits/be1e7b55144fa7b6c94e085779b56ad86ea62f6d